go/contacts/cache.go

```diff
@@ -217,8 +223,10 @@ func (c *CachedContactsProvider) LookupAll(mctx libkb.MetaContext, emails []keyb
 	mctx.Debug("After checking cache, %d emails and %d numbers left to be looked up", len(remainingEmails), len(remainingNumbers))
 
 	if len(remainingEmails)+len(remainingNumbers) > 0 {
-		apiRes, err := c.Provider.LookupAll(mctx, remainingEmails, remainingNumbers, userRegion)
+		apiRes, err := c.Provider.LookupAllWithToken(mctx, remainingEmails, remainingNumbers, userRegion, conCache.Token)
 		if err == nil {
+			conCache.Token = apiRes.Token
```

Comment:
looks like the adhoc tofu searches for just one phonenumber/email will always overwrite your big tokens for contact lists

Reply:
I don't think so, the server sends back a token for everything in the token as well as what was requested

Reply:
sorry I could be wrong about that...but if the server did do that, it'd be ok?